index.html

```diff
@@ -6,5 +6,21 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Wireframe 3</title>
   </head>
-  <body></body>
+  <body>
+    <nav class="navbar">
+      <img src="assets/img/logo.svg" alt="">
```

Comment:
I overlooked that alt is empty, it's good to add some alt text, `alt="logo"` in this case